tortoise/models.py

```diff
@@ -724,6 +741,47 @@ def update_from_dict(self, data: dict) -> MODEL:
         self._set_kwargs(data)
         return self  # type: ignore
 
+    @classmethod
+    def register_listener(cls, signal: Signals, listener: Callable):
+        assert callable(listener), "listener must be callable!"
```

Comment:
Could you change this to an `if` that raises `tortoise.exceptions.ConfigurationError`?

Reply:
yeah,that's better.